src/kernel-install/kernel-install.c

```diff
@@ -1294,7 +1294,7 @@ static int verb_add_all(int argc, char *argv[], void *userdata) {
         }
 
         if (n > 0)
-                log_info("Installed %zu kernels.", n);
+                log_debug("Installed %zu kernel%s.", n, (n>1?"s":"")); // XXX: use ngettext ffs
```

Comment:
Please add proper space

Reply:
this is a debug message, does the plural really matter?

maybe just do `kernel(s)`?